lib/engine.py

```diff
@@ -379,20 +379,13 @@ def database_name(self, name=None):
 
         return db_name
 
-    def download_file(self, url, filename, clean_line_endings=True):
+    def download_file(self, url, filename):
         """Downloads a file to the raw data directory."""
         if not self.find_file(filename):
             path = self.format_filename(filename)
             self.create_raw_data_dir()
             print "Downloading " + filename + "..."
-            file = urllib.urlopen(url)
-            local_file = open(path, 'wb')
-            if clean_line_endings and (filename.split('.')[-1].lower() not in ["exe", "zip", "xls"]):
-                local_file.write(file.read().replace("\r\n", "\n").replace("\r", "\n"))
```

Comment:
Thanks @Deborah-Digges, so here we are reading the entire file which is bad, to have a step wise change I suggest first a few ways to go about this,
1) use the old method to download the file and use the new method to download the same file. try to analyze both files and see what is happening.
2) you could also change the file.read() to reading line by line or a chunk read(size) and reprocessing. Travis is a linux based platform so we should not be having much of the problems comparing to windows platforms.